test/citestutils.js

```diff
@@ -62,6 +63,24 @@ function testBackup(params, databaseName, outputStream, callback) {
     }
   }
 
+  // Pipe via encryption if requested
+  if (params.encryption) {
+    if (params.useApi) {
+      // Currently only CLI support for testing encryption
+      callback(new Error('UNSUPPORTED: cannot abort API backups at this time.'));
```

Comment:
Fixed in e0246b5
Chose to go with `Not implemented` and `cannot test` to avoid anyone thinking that it wasn't supported by the API, which it is, just harder to test because of needing to use a `crypto` lib or similar to pipe the stream.